Tasks/VsTest/task.json

```diff
@@ -209,18 +224,15 @@
             "groupName": "reportingOptions"
         }
     ],
-    "instanceNameFormat": "Test Assemblies $(testAssembly)",
+    "instanceNameFormat": "Test Assemblies",
     "execution": {
+        "Node": {
+            "target": "vstest.js"
+        },
         "PowerShell": {
             "target": "$(currentDirectory)\\VSTest.ps1",
             "argumentFormat": "",
-            "workingDirectory": "$(currentDirectory)",
-            "platforms": [
-                "windows"
-            ]
-        },
-        "Node": {
-            "target": "vstest.js"
+            "workingDirectory": "$(currentDirectory)"
```

Comment:
kept it as an option for user to switch to powershell by updating task.json. We can remove it when we remove other ps scripts